test/docstore.test.js

```diff
@@ -3,160 +3,169 @@
-      await db.put(doc1)
-      await db.put(doc2)
-      await db.put(doc3)
-      const value = db.get('hello')
-      assert.deepEqual(value, [doc1, doc2])
-    })
-
-    it('get after delete', async () => {
-      const doc1 = { _id: 'hello world', doc: 'some things'}
-      const doc2 = { _id: 'hello universe', doc: 'all the things'}
-      const doc3 = { _id: 'sup world', doc: 'other things'}
-      await db.put(doc1)
-      await db.put(doc2)
-      await db.put(doc3)
-      await db.del('hello universe')
-      const value1 = db.get('hello')
-      const value2 = db.get('sup')
-      assert.deepEqual(value1, [doc1])
-      assert.deepEqual(value2, [doc3])
-    })
-
-    it('put updates a value', async () => {
-      const doc1 = { _id: 'hello world', doc: 'all the things'}
-      const doc2 = { _id: 'hello world', doc: 'some of the things'}
-      await db.put(doc1)
-      await db.put(doc2)
-      const value = db.get('hello')
-      assert.deepEqual(value, [doc2])
-    })
-
-    it('query', async () => {
-      const doc1 = { _id: 'hello world', doc: 'all the things', views: 17}
-      const doc2 = { _id: 'sup world', doc: 'some of the things', views: 10}
-      const doc3 = { _id: 'hello other world', doc: 'none of the things', views: 5}
-      const doc4 = { _id: 'hey universe', doc: ''}
-
-      await db.put(doc1)
-      await db.put(doc2)
-      await db.put(doc3)
-      await db.put(doc4)
-
-      const value1 = db.query((e) => e.views > 5)
-      const value2 = db.query((e) => e.views > 10)
-      const value3 = db.query((e) => e.views > 17)
-
-      assert.deepEqual(value1, [doc1, doc2])
-      assert.deepEqual(value2, [doc1])
-      assert.deepEqual(value3, [])
-    })
-
-    it('query after delete', async () => {
-      const doc1 = { _id: 'hello world', doc: 'all the things', views: 17}
-      const doc2 = { _id: 'sup world', doc: 'some of the things', views: 10}
-      const doc3 = { _id: 'hello other world', doc: 'none of the things', views: 5}
-      const doc4 = { _id: 'hey universe', doc: ''}
-
-      await db.put(doc1)
-      await db.put(doc2)
-      await db.put(doc3)
-      await db.del('hello world')
-      await db.put(doc4)
-      const value1 = db.query((e) => e.views >= 5)
-      const value2 = db.query((e) => e.views >= 10)
-      assert.deepEqual(value1, [doc2, doc3])
-      assert.deepEqual(value2, [doc2])
-    })
-  })
+    after(async () => {
+      if(orbitdb1) 
+        await orbitdb1.stop()
 
-  describe('Specified index', function() {
-    beforeEach(async () => {
-      const options = { 
-        indexBy: 'doc', 
-        replicate: false, 
-        maxHistory: 0 
-      }
-      db = await orbitdb1.docstore(config.dbname, options)
+      if (ipfsd) 
+        await stopIpfs(ipfsd)
     })
 
-    afterEach(async () => {
-      await db.drop()
+    it('creates and opens a database', async () => {
+      db = await orbitdb1.docstore('first doc database')
```

Comment:
Duplicate?

Reply:
Good catch and it seems to be a problem with all store. They also don't seem to contain an assert for that test.